lib/matplotlib/dates.py

```diff
@@ -83,10 +83,11 @@
      Out[1]: 732401
 
 All the Matplotlib date converters, tickers and formatters are timezone aware.
-If no explicit timezone is provided, :rc:`timezone` is assumed.  If you want to
-use a custom time zone, pass a `datetime.tzinfo` instance with the tz keyword
-argument to `num2date`, `.Axis.axis_date`, and any custom date tickers or
-locators you create.
+If no explicit timezone is provided, :rc:`timezone` is assumed, provided as a
+string.  If you want to use a custom timezone, pass a `datetime.tzinfo`
+instance or a string with the timezone name with the tz keyword argument to
```

Comment:
```suggestion
instance or a string with the timezone name to the tz keyword argument to
```

Reply:
Should it be "to the ... to"?

To me maybe "to the ... of" is better (but I agree that "with ... with" is not a good option).

Reply:
Perhaps "...as the tz keyword argument..." is better?

Reply:
There was a suggestion from jklymak (I do not tag to maybe reduce the number of notifications) where this was split into several sentences. See below. I think it worked out nicely.